packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -119,91 +123,85 @@ export default class AwsS3Multipart extends BasePlugin {
     this.#client = new RequestClient(uppy, opts)
 
     const defaultOptions = {
-      timeout: 30 * 1000,
+      timeout: 30 * 1000, // todo this in no longer in use. either implement or remove.
       limit: 0,
-      retryDelays: [0, 1000, 3000, 5000],
+      retryDelays: [0, 1000, 3000, 5000], // todo this in no longer in use. either implement or remove.
       createMultipartUpload: this.createMultipartUpload.bind(this),
       listParts: this.listParts.bind(this),
-      prepareUploadParts: this.prepareUploadParts.bind(this),
+      prepareUploadParts: this.prepareUploadParts.bind(this), // todo this in no longer in use. either implement or remove.
```

Comment:
We probably need to first deprecate it before getting rid of it completely.

Reply:
That means we have to implement an alternative flow that uses the option, right? Currently it doesn't get called at all, so we have already effectively gotten rid of it

Reply:
Yes correct